### PR TITLE
Migrate from CentOS to Alma Linux 9

### PR DESCRIPTION
**Pull request by Praveen2112**

### CODE REVIEW

**Review by kokosing — APPROVED** [review]

**Comment by Praveen2112** [comment]
The PR is not ready

**Comment by Praveen2112** [comment]
I think we fixed it for all images - Will cleanup this code a bit

**Review by wendigo — APPROVED** [review]

**Comment by wendigo** [comment]
@Praveen2112 ship it

**Comment by Praveen2112** [comment]
I'll do some cleanup and merge it. Thanks for the review

**Comment by Praveen2112** [comment]
The hive failure looks like a legit one

**Comment by Praveen2112** [comment]
Running PTs for additional validation - Found a few failures - fixing them.

**Review by kokosing — APPROVED** [review]
Thank you!

File: testing/hdp3.1-hive/Dockerfile [hunk 1]
```diff
@@ -17,6 +17,10 @@ FROM ghcr.io/trinodb/testing/hdp3.1-hive:38
 
 COPY ./files /
```
[the remark this answers is not in the document]
Comment (Praveen2112) [reply testing/hdp3.1-hive/Dockerfile:18]: We use them for `HiveHadoop` image

File: testing/phoenix5/Dockerfile [hunk 2]
```diff
@@ -11,7 +11,7 @@
 # limitations under the License.
```
[the remark this answers is not in the document]
Comment (Praveen2112) [reply testing/phoenix5/Dockerfile:11]: We still use them for phoenix connector

**Pull request merged**